src/components/application-loader/initializers/fetch-motd.ts

```diff
@@ -47,7 +48,7 @@ export const fetchMotd = async (customizeAssetsUrl: string): Promise<void> => {
 
   const motdText = await response.text()
 
-  const lastModified = response.headers.get('Last-Modified')
+  const lastModified = response.headers.get('Last-Modified') || response.headers.get('etag')
   if (!lastModified) {
     log.warn("'Last-Modified' not found for motd.txt!")
```

Comment:
Maybe adjust this message

```suggestion
    log.warn("'Last-Modified' or 'Etag' not found for motd.txt!")
```